src/library/Uno.Themes.WinUI.Markup/Theme/ComboBox.cs

```diff
@@ -231,25 +210,22 @@ public static class PillFill
 
 				public static class DropDown
 				{
-					public static class GlyphForeground
+					public static class Foreground
 					{
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownForeground")]
+						public static ResourceValue<Brush> Default => new("ComboBoxDropDownForeground", true);
+
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownGlyphForeground")]
-						public static ResourceValue<Brush> Default => new("ComboBoxDropDownGlyphForeground", true);
+						public static ResourceValue<Brush> Glyph => new("ComboBoxDropDownGlyphForeground", true);
 
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownGlyphForegroundFocusedPressed")]
-						public static ResourceValue<Brush> FocusedPressed => new("ComboBoxDropDownGlyphForegroundFocusedPressed", true);
+						public static ResourceValue<Brush> GlyphFocusedPressed => new("ComboBoxDropDownGlyphForegroundFocusedPressed", true);
 
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownGlyphForegroundFocused")]
-						public static ResourceValue<Brush> Focused => new("ComboBoxDropDownGlyphForegroundFocused", true);
+						public static ResourceValue<Brush> GlyphFocused => new("ComboBoxDropDownGlyphForegroundFocused", true);
 
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownGlyphForegroundDisabled")]
-						public static ResourceValue<Brush> Disabled => new("ComboBoxDropDownGlyphForegroundDisabled", true);
-					}
-
-					public static class Foreground
-					{
-						[ResourceKeyDefinition(typeof(Brush), "ComboBoxDropDownForeground")]
-						public static ResourceValue<Brush> Default => new("ComboBoxDropDownForeground", true);
+						public static ResourceValue<Brush> GlyphDisabled => new("ComboBoxDropDownGlyphForegroundDisabled", true);
 					}
 
 					public static class BorderBrush
```

Comment:
Convert to a direct property on the `DropDown` class since the `BorderBrush` has no visual state variants here